features/aggregation.feature

```diff
@@ -138,3 +138,76 @@ Feature: Aggregated Data Pages
+    And the search controller is running
+    When I navigate to "/testpath/publications"
+    Then the page should have the following content
+    """
+        {
+            "#main h1": "Page not found"
+        }
+    """
+
+  @agg_rss
+  Scenario: GET subtopic pre-filtered page with matching topic
+    Given there is a Search API that gives a successful response and returns 10 results
+    And there is a Topic API that returns the "economy" topic and the "environmentalaccounts" subtopic
+    And the search controller is running
+    When I navigate to "/economy/environmentalaccounts/publications"
+    Then the page should have the following content
+    """
+        {
+            "#main h1": "Publications related to environmental accounts",
+            ".search__count h2": "10 results"
+        }
+    """
+
+  @agg_rss
+  Scenario: GET subtopic pre-filtered page with non-matching topic
+    Given there is a Search API that gives a successful response and returns 10 results
+    And there is a Topic API that returns the "economy" topic and the "environmentalaccounts" subtopic
+    And the search controller is running
+    When I navigate to "/economy/testtopic/publications"
+    Then the page should have the following content
+    """
+        {
+            "#main h1": "Page not found"
+        }
+    """
+
+@agg_rss
+  Scenario: GET rss for subtopic pre-filtered page with matching topic
+  Given there is a Search API that gives a successful response and returns 10 results
+  And there is a Topic API that returns the "economy" root topic and the "environmental" subtopic for requestQuery "rss"
+  And the search controller is running
+  When I navigate to "/economy/environmental/publications?rss"
+  Then the page should have the following xml content
+    """
+      <?xml version="1.0" encoding="ISO-8859-1"?>
+      <rss xmlns:dc="http://purl.org/dc/elements/1.1/" version="2.0">
+      <channel>
+        <title>Latest ONS releases matching (topic: /economy/environmental, type: data).</title>
+        <link>http://dp.aws.onsdigital.uk/economy/environmental?rss</link>
+        <description>Latest ONS releases</description>
+        <category>/economy/environmental</category>
+        <dc:subject>/economy/environmental</dc:subject>
+      </channel>
+      </rss>
+    """
+  And the response header "Content-Type" should contain "application/rss+xml; charset=ISO-8859-1"
+
+@agg_rss
+  Scenario: GET RSS subtopic pre-filtered page with non-matching topic
+    Given there is a Search API that gives a successful response and returns 10 results
+    And there is a Topic API that returns the "economy" root topic and the "environmental" subtopic for requestQuery "rss"
+    And the search controller is running
+    When I navigate to "/economy/testtopic/publications/rss"
+    Then the page should have the following content
+    """
+        {
+            "#main h1": "Page not found"
+        }
+    """
```

Comment:
Please add a new line at the end of this file.

Reply:
@AWitcherONS Good opportunity to squash commits